subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -33,6 +33,18 @@ Some plugins will break with this new version of Gradle, for example because the
 . Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
+[[changes_6.8]]
+== Upgrading from 6.7
+
+=== Potential breaking changes
+
+==== Eclipse plugin has updated classpath
+
+Previously, the default Eclipse classpath was calcuated from a fixed set of dependency configurations: `compileClasspath`, `runtimeClasspath`,`testCompileClasspath`, and `testRuntimeClasspath`.
+
+Since Gradle 6.8, the the default classpath automatically contains all `xxxCompileClasspath` and `xxxRuntimeClasspath` for every custom
+source sets defined by the build.
```

Comment:
```suggestion
Since Gradle 6.8, projects imported into Eclipse include the compile and runtime classpath for every source set defined by the build.
```